server/api/config.go

```diff
@@ -286,6 +286,7 @@ func getConfigMap(cfg map[string]interface{}, key []string, value interface{}) m
 	return cfg
 }
 
+<<<<<<< HEAD
```

Comment:
```suggestion
<<<<<<< HEAD

```

Plz fix  the conflict.